### ISSUE TITLE
De-vendor meshcat dependency

### ISSUE THREAD

**Issue by traversaro** [conda-forge/meshcat-python-feedstock#1]
Not sure if possible or desirable, but let's keep track of it. See https://github.com/conda-forge/staged-recipes/pull/14312#issuecomment-803295967 .

**Comment by danzimmerman** [comment]
@traversaro Maybe relevant to whether it's desirable, I needed to modify `index.js` with changes from [meshcat/pulls/99](https://github.com/rdeits/meshcat/pull/99/) and rebuild `main.min.js` to get Conda-installed `meshcat-python 0.3.1` to work properly for me:

https://github.com/rdeits/meshcat-python/issues/84#issuecomment-953947232

**Comment by traversaro** [comment]
Yes, this is definitely a good example in which if we packaged meshcat separately we would just need to update meshcat, instead of waiting for meshcat-python to update its own vendored meshcat. 

In the meanwhile, if you can think of any way of patching the existing meshcat-python to use the new meshcat, feel free to open a PR.
